### ISSUE TITLE
Water blocks all radiation from above

### ISSUE THREAD

**Issue by Nick1st** [Somdudewillson/nc-environmental-rads#3]
When standing under a water-source or flowing block all sky radiation is blocked. That behavior is Water only. (I tried Lava and did not get the same results.)

**Comment by Nick1st** [comment]
![2021-03-27_17 34 52](https://user-images.githubusercontent.com/54022594/112727469-eb0ba100-8f22-11eb-95c4-81341017d16a.png)

![2021-03-27_17 35 46](https://user-images.githubusercontent.com/54022594/112727477-ee069180-8f22-11eb-8d4f-f250cd5059fc.png)
![2021-03-27_17 35 50](https://user-images.githubusercontent.com/54022594/112727479-ee9f2800-8f22-11eb-917f-e741686417f3.png)

**Comment by DanteMaxwell** [comment]
So, super old, but in case you never got your answer, that's because the default value of radiation absorption of blocks is = 0.01 * x (where x is = to the block's Hardness). Water has a Hardness of 100, meaning 0.01 * 100 = 1.0, or 100% of radiation.

Supposedly, there's a way to customize and override the Hardness calculation in the configs, but I have yet to succeed in getting things like Minecraft's water or Thermal Foundation or Immersive Engineering's blocks to work with this. Chisel, on the other hand, works perfectly fine; so long as you input EVERY. SINGLE. VARIANT. of the target block into the config.

**Comment by DanteMaxwell** [comment]
I have subsequently figured out how to resolve all remaining issues. Many mods, and even Minecraft proper, need to use TILES instead of blocks.

To resolve Thermal Foundation, for example, lead blocks, it goes like this:
`D:"thermalfoundation:storage.lead.name"=1.0`

Immersive Engineering, like this (example: Leaded Concrete):
`D:"immersiveengineering:stone_decoration.concrete_leaded"=1.0`

And finally, to answer your original request from nearly a year and a half ago, here's how to fix Water:
`D:tile.water=0.25`

**Comment by Nick1st** [comment]
Yeah, I don't need it anymore, but thanks anyway. I'm sure this will help others at least.
